msvc/msvc-disabled-warnings.h

```diff
@@ -4,31 +4,31 @@
 
 // FIXME This is all questionable but the logs are flooded and nothing else is fixing them.
 #define _CRT_SECURE_NO_WARNINGS 1
-#pragma warning(disable:4013) // function undefined; assuming extern returning int
+#pragma warning(  error:4013) // function undefined; assuming extern returning int
```

Comment:
nit: shouldn't these go into a different file not called `msvc-disabled-warnings.h` now?

Reply:
msvc-enabled-errors? I guess so.
Or rename this file to msvc-warnings.h so its content can vary?

Reply:
I don't have a preference so let's wait for @lateralusX opinion.

Reply:
Mono is generally anti-churn, but renaming it to something less specific would fit as its contents change.

In Windows, warning.h, which is force-included from command line, so early in every compile, gradually morphed just to be "early stuff", but also a mix of lines like this.

mono-warning.h?
It could morph to cover clang and gcc.